packages/electron-app/src/main.js

```diff
@@ -1,10 +1,7 @@
 const { app, BrowserWindow } = require('electron');
 const path = require('path');
 
-// Handle creating/removing shortcuts on Windows when installing/uninstalling.
-if (require('electron-squirrel-startup')) {
-  app.quit();
-}
+import handleSquirrelEvent from 'utils/handleSquirrelEvent';
```

Comment:
I converted the handleSquirrelEvent to a Typescript file but reverted it to plain 'ol JS for this